File: Software Technologies/Shopping-List-master/JavaScript Solution/models/product.js
const Sequelize = require('sequelize');

module.exports = function(sequelize){

    const Product = sequelize.define("Product", {
            priority: {
                type: Sequelize.INTEGER,
                required: true,
                allowNull: false
            },
            name: {
                type: Sequelize.TEXT,
                required: true,
                allowNull: false
            },
            quantity: {
                type: Sequelize.INTEGER,
                required: true,
                allowNull: false
            },
            status: {
                type: Sequelize.TEXT,
                required: true,
                allowNull: false
            }
    }, {
        timestamps: false
    });

    return Product;
};

File: 03.Software Technologies/Shopping-List-master/CSharp/ShoppingList/Models/Product.cs
namespace ShoppingList.Models
{
    using System.ComponentModel.DataAnnotations;

    public class Product
    {
        [Key]
        public int Id { get; set; }

        public int Priority { get; set; }

        [Required]
        public string Name { get; set; }

        public int Quantity { get; set; }

        [Required]
        public string Status { get; set; }

    }
}
